src/tokenizer.cpp

```diff
@@ -939,6 +940,20 @@ gb_internal void tokenizer_get_token(Tokenizer *t, Token *token, int repeat=0) {
 			if (t->curr_rune == '!') {
 				token->kind = Token_Comment;
 				tokenizer_skip_line(t);
+			} else if (t->curr_rune == '+') {
+				token->kind = Token_FileTag;
+				
+				// Skip the line or until it ends or until we hit was is probably a comment.
+				// The parsing of tags happens in `parse_file`.
+				while (t->curr_rune != GB_RUNE_EOF) {
+					if (t->curr_rune == '\n') {
+						break;
+					}
+					if (t->curr_rune == '/') {
+						break;
```

Comment:
This makes it stop at a _probable_ comment. Could I perhaps somehow peek at the next rune to make sure it's `/` or `*`?

Reply:
`peek_byte`? Not rune, but for `/` and `*` it shouldn't matter.

Reply:
Thanks. I think I can leave it as it is. There are no file tags with `/` in them anyways, so I can just always abort the loop on the first slash. If we do some more proper  tokenization of tags at some point later we can do something proper for this.